plugins/for_each_line.cc

```diff
@@ -291,7 +286,7 @@ void forEachLineBlock(std::istream & stream,
             bool lastBlock = false;
 
             try {
-                if (mapped) {
+                if (mapped && false) {
```

Comment:
That is MLDB-1426.  Since the code is currently buggy, we should apply this hunk as a separate PR to work around MLDB-1426 while we await a proper fix.